renderer/components/Pay/PayPanelFooter.js

```diff
@@ -100,14 +98,13 @@ const PayPanelFooter = props => {
 
   // Determine which buttons should be visible.
   const hasBackButton = currentStep !== PAY_FORM_STEPS.address
-  const hasSubmitButton = currentStep !== PAY_FORM_STEPS.address || (isOnchain || isLn)
 
   return (
     <Flex flexDirection="column">
       {renderLiquidityWarning(props)}
       <PayButtons
         hasBackButton={hasBackButton}
-        hasSubmitButton={hasSubmitButton}
+        hasSubmitButton
```

Comment:
This seems wrong. The next button is never supposed to be visible on the first step of the pay form since the form automatically moves you to the next step after entering a valid address.

Reply:
what if you press back?

Reply:
Ok, so perhaps we should make it visible only after going back?

Right now you have to edit the value for it to pick up the change and move you forwards

Reply:
Not sure i understand. If you don't hit back then you can't see  it because you can't see that  form

Reply:
When the form first loads, the button should not be visible because the form is empty. Then you enter an address and it automatically moves you forwards to the next step. If you go back then the button could show in order to address the issue you are observing with the button not being visible when you go back.

Reply:
I see makes sense. Still maybe this is not actually desired. To my knowledge this is the only place where we would hide the button. In all other places we usually allow user to do one attempt and then do inline validation. I understand that here we have some sort of automatic transition, but i'm not sure this is enough to justify hiding of the button. What do u think

Reply:
That's true. I think that the thought process here was that there would be no possible reason for anyone to do anything on this form other than to paste an address, so why clutter the ui with distractions? Instead, lets make the page as basic and single-purpose as possible. Not that a submit button is probably too much of a distraction, but that was the logic.

I think the best thing would be to keep it hidden if the user is new to the form in order to keep the existing behaviou as designed, and show it if the user is coming back from another form step